dev-env/bin/daml-docker-head

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+set -eu
+
+version=0.0.0
+sdk_tarball=sdk-release-tarball.tar.gz
+
+# Make sure they are in the right repo
+readonly repo_root="$(bazel info workspace)"
+if [[ "${repo_root}/dev-env/bin/daml-docker-head" != "${BASH_SOURCE[0]}" ]]; then
+  echo "Repo root does not match the script being run."
+  echo "Make sure you run this script from a directory within the 'daml' repository "
+  exit 1
+fi
+
+# Make sure they have docker
+if ! which docker > /dev/null ; then
+  echo "docker does not seems to be installed."
+  echo "See https://docs.docker.com/engine/install/ for more detail about docker installation."
+  exit 1
+fi
+
+tmpdir=$(mktemp -d -t docker-daml-XXXXXXXXXX)
+
+bazel build //release:sdk-release-tarball
+
+cp ${repo_root}/bazel-bin/release/${sdk_tarball} ${tmpdir}/
```

Comment:
perhaps worth adding a comment that we need this because docker really doesn’t like `COPY`ing from files outside of the build directory and it also doesn’t like symlinks.